modules/ROOT/pages/backup-restore/aggregate.adoc

```diff
@@ -205,3 +207,40 @@ bin/neo4j-admin backup aggregate --from-path=azb://myStorageAccount/myContainer/
 ----
 ======
 =====
+
+
+[[aggregate-unrecovered-full-backup]]
+=== Aggregating an unrecovered full backup
+
+To check if a full backup is in a recovered state, run the `neo4j-admin backup inspect` command:
+
+[source,shell]
+----
+bin/neo4j-admin backup inspect /mnt/backups/neo4j-2025-09-10T08-14-56.backup
+----
+
+The output will include the `RECOVERED` column.
+If it shows `false`, the backup is not yet recovered:
+
+[output]
+----
+|                                                 FILE | DATABASE |                          DATABASE ID |          TIME (UTC) | FULL | COMPRESSED | LOWEST TX | HIGHEST TX | STORE ID HASH | RECOVERED |
+| file:///mnt/backups/neo4j-2025-09-10T08-14-56.backup |    neo4j | 4e98fab8-6ae6-46cc-84c0-61ee8a8db60a | 2025-09-10T08:14:56 | true |       true |         1 |          3 |    -116350002 |     false |
+----
+
+If your full backup is not recovered, you have to aggregate it first.
```

Comment:
I'm not keen on the 'have to' here. An unrecovered backup is still usable in most situations, such as `restore`. Perhaps:
```suggestion
If your full backup is not recovered, aggregating it will perform recovery.
```